concent_api/concent_api/settings/base.py

```diff
@@ -32,6 +32,7 @@
 ]
 
 MIDDLEWARE = [
+    'concent_api.middleware.HandleServerErrorMiddleware',
```

Comment:
Could you add a comment explaining that we're disabling this middleware in `testing.py`? It's not necessarily something that belongs here but I don't know of a better place to put it and it's something that someone tinkering with our middleware settings should be aware of.

Reply:
done